### DISCUSSION TITLE
Difference Between Versions at Hugging Face?

### DISCUSSION THREAD

**Discussion by DavidDragonsage** [metercai/SimpleSDXL#66] [Q&A] [answered]
At https://huggingface.co/metercai/SimpleSDXL2/tree/main there are two versions of the installer:
SimpleSDXL2_win.exe.7z0820 and SimpleSDXL2_win2.exe.7z0820
what is the difference and which is the correct version?

I installed using SimpleSDXL2_win2.exe. Today, one of my colleagues said that SimpleSDXL2_win.exe did not work.

Also, what are SimpleSDXL2_win_dev.exe.7z0812_2 and SimpleSDXL2_win_dev2.exe.7z0812_4 ?

**Comment by metercai** [comment]
Their built-in Pytorch+CUDA versions are different. The version of dev2 will be lower, mainly for compatibility reasons.

**Reply by DavidDragonsage** [reply]
I'm sorry, I don't quite understand. So "dev2" corresponds to SimpleSDXL2_win2.exe? And SimpleSDXL2_win2.exe is more compatible?

So we should be using SimpleSDXL2_win2.exe to have the greatest compatibility with mainline Fooocus?

**Reply by metercai** [reply]
> I'm sorry, I don't quite understand. So "dev2" corresponds to SimpleSDXL2_win2.exe? And SimpleSDXL2_win2.exe is more compatible?
> 
> So we should be using SimpleSDXL2_win2.exe to have the greatest compatibility with mainline Fooocus?

It is misunderstanding. It is hardware compatibility.
Prioritize using win or win_dev. If there are hardware compatibility issues such as lagging in generating, you can try switching to win2 or win_dev2, and the problem may disappear.

**Reply by DavidDragonsage** [reply]
Thank you, I understand.

And what is the difference between "win" and "win_dev"?

**Reply by metercai** [reply, marked as answer]
> Thank you, I understand.
> 
> And what is the difference between "win" and "win_dev"?

win = Stable version on Windows
win_dev = Development and testing version on Windows

**Reply by DavidDragonsage** [reply]
Thank you so much! This is great information and I will pass it on. 👍
